tools/hangar/src/cli.bench.ts

```diff
@@ -0,0 +1,34 @@
+import { basename } from "path";
+import { bench, describe } from "vitest";
+import { runWingCommand } from "./utils";
+import { benchmarksTestDir, walkdir } from "./paths";
+
+const targets = ["sim", "tf-aws"];
+const files: string[] = [];
+for await (const wingFile of walkdir(benchmarksTestDir)) {
+  if (wingFile.endsWith(".w")) {
+    files.push(wingFile);
+  }
+}
+
+describe("compile", async () => {
```

Comment:
Do we have a way to record thresholds per test and fail if the test exceeds the threashold?

Reply:
Not yet, would have to be a post-process. Should that be a blocker for this PR?

Reply:
Perhaps we can just configure the timeout for each test as the threshold and simply have a map in the test code that maps test files to their timeouts.

Reply:
Ok I added a system where you can add a comment to the wing file to designate this info and it'll fail the test based on that.